### PR TITLE
Added email templates and methods to email client class

### PR DESCRIPTION
**Pull request by jellyyams**
This was a struggle.

### CODE REVIEW

File: blueprints/public.py [hunk 1]
```diff
@@ -135,11 +138,12 @@ def confirmation():
 @public.route("/export/<eventid>", methods=["GET"])
 def export_event(eventid):
     if request.method == "GET":
-        event_data = db.events.find_one({"_id": ObjectId(eventid)})
+        event_data = db.get_one(ObjectId(eventid))
+        email.send_edit_link(request.base_url, event_data)
```
Comment (jellyyams) [inline blueprints/public.py:142]: forgot to delete this line

File: requirements.txt [hunk 2]
```diff
@@ -26,10 +26,11 @@ PyYAML==5.3.1
 requests==2.23.0
 requests-oauthlib==1.3.0
 rsa==4.0
-sendgrid==6.3.1
+sendgrid==6.4.3
 six==1.14.0
 soupsieve==2.0
 SQLAlchemy==1.3.16
+starkbank-ecdsa==1.0.0
```
Comment (jack-greenberg) [inline requirements.txt:33]: Did you add this? If so, why?

File: static/js/components/event-page.js [hunk 3]
```diff
@@ -1,6 +1,13 @@
 import React from 'react';
 import client from '../api';
+<<<<<<< HEAD
+=======
 
+<<<<<<< HEAD
+>>>>>>> Added routes for edit, add, confirmation, and admin
+
+=======
+>>>>>>> fixed typos/bugs that jack pointed out
```
Comment (jack-greenberg) [inline static/js/components/event-page.js:10]: :( merge conflict not resolved

File: static/js/components/event-page.js [hunk 4]
```diff
@@ -93,10 +105,14 @@ export default class EventPage extends React.Component {
...
                       </td>
                       <td class="Event__content__text">
+<<<<<<< HEAD
                         <span class="Event__content__date">
                           {this.props.event.start.toLocaleDateString("en-US", { weekday: 'long', year: 'numeric', month: 'long', day: 'numeric' })}
                         </span>
                         |
+=======
+                     
+>>>>>>> in the middle of osmething
```
Comment (jack-greenberg) [inline static/js/components/event-page.js:115]: mergesssss

File: blueprints/public.py (hunk 1 above)
Comment (jack-greenberg) [reply blueprints/public.py:142]: ok plz delete when u make changes

File: requirements.txt (hunk 2 above)
Comment (jellyyams) [reply requirements.txt:33]: nope didn't add this. if I did I didn't mean to. Ill remove it

**Pull request merged**